test/outdated/goldens/helptext.txt

```diff
@@ -0,0 +1,22 @@
+$ pub outdated --help
+Analyze your dependencies to find which ones can be upgraded.
+
+Usage: pub outdated [options]
+-h, --help                         Print this usage information.
+    --json                         Outputs the results in a json formatted
+                                   report.
```

Comment:
```suggestion
    --json                         Output the results in JSON format.
```

When the helps starts with a verb, it should be consistently either 3rd person (with an 's' at the end, generally) or 2nd person (no 's'). Most of these commands use 2nd person, so let's do that here.

Reply:
Done